COLLABORATOR_GUIDE.md

```diff
@@ -240,14 +240,48 @@ You can find more information [in the full LTS plan](https://github.com/nodejs/l
 
 #### How does LTS work?
 
-Once a stable branch enters LTS, no new features may be added to that release. Changes are
-limited to bug fixes, security updates, possible npm updates, documentation updates, and certain
-performance improvements that can be demonstrated to not break existing applications.
-Semver-minor changes are only permitted if required for bug fixes. Semver-major changes are only
-permitted if required for critical security and bug fixes.
-
-Once a stable branch moves into Maintenance mode, only **critical** bugs, **critical** security fixes,
-and documentation updates will be permitted.
+Once a stable branch enters LTS, no new features may be added to that release.
+Changes are limited to bug fixes, security updates, possible npm updates,
+documentation updates, and certain performance improvements that can be
+demonstrated to not break existing applications. Semver-minor changes are only
+permitted if required for bug fixes and then only on a case-by-case basis with
+LTS WG and possibly Core Technical Committee (CTC) review. Semver-major changes
+are permitted only if required for security related fixes.
+
+Once a stable branch moves into Maintenance mode, only **critical** bugs,
```

Comment:
We don't, but it may be useful to try to formalize it a bit more. Obviously, it doesn't have to be set in stone, and it doesn't have to be formalized before landing this PR.

We might want to formalize this in a way similar than [how it was formalized in joyent/node](https://github.com/nodejs/nodejs.org-archive/pull/111/files#diff-6ebcbf95afb7c6f9b940dfcb5ee35122R282).